pages/Addons.md

```diff
@@ -76,5 +76,5 @@ Here is a list of all Addon Jams that have been hosted so far:
 ### Creating your own Addons?
 Do you want to create your own Slimefun Addon? Check out this [Developer Guide](https://github.com/Slimefun/Slimefun4/wiki/Developer-Guide).
 
-If you have made an Addon for Slimefun and want it to be listed on here, then just create an Issue and select the "Addition" template:
-https://github.com/TheBusyBiscuit/Slimefun4-Wiki/issues
+If you have made an Addon for Slimefun and want it to be listed on here, simply make a new pull request on our wiki repository. You can find a tutorial here:
+https://github.com/Slimefun/Slimefun4/wiki/Expanding-the-Wiki
```

Comment:
While the pull request is the preferred method, it may be wise to still mention the option of creating an issue for it if a pull request is not possible.